docs/api-testing/project-access.md

```diff
@@ -1,66 +1,85 @@
+- **Organization Admin**: Organization Admins can create and interact with all data in all projects across their organization.
 
-You cannot edit project access once it has been set.
+:::tip
+You can view the access set for projects on the Projects page.<br/><img src={useBaseUrl('img/api-fortress/2021/12/accessOrg2.png')} alt="New user UI" width="500" />
+:::
 
-#### User Scenarios
+## Create a Project
 
-- **New User** (no team assignments or projects): When this user first logs in to API Testing, they will be guided through creating a project and a test.<br/><img src={useBaseUrl('img/api-fortress/2021/12/emptyState.png')} alt="New user UI" width="500" /><br/> They will have the option to set organization-wide project access and will only be able to see the projects they created.
-- **Team Member**: This user belongs to a team and can create and interact with their team's projects and organization-wide projects. They cannot view projects with access restricted to an outside team. When creating a project, they will have the option to set access to be team-specific and organization-wide.
-- **Organization Admin**: This user can create and interact with all data in all projects across their organization.
+You create projects and edit project settings on the **Settings > General > Project details** page.  
 
-:::note
-You can also view access settings from your projects page.<br/><img src={useBaseUrl('img/api-fortress/2021/12/accessOrg2.png')} alt="New user UI" width="500" />
-:::
+1. In **API Testing**, on the **Projects** page, click **Create Project**. 
+1. Select an option in the **Create from** dropdown. 
+1. Optionally, select one or more **Tags**, or click **Add custom tag** to add a new tag. 
+1. Optionally, enter a **Description** of the project. 
+1. Optionally, enter **Notes** about the project. 
+1. Select the **Access** level for the project. If teams have been set up for your organization and you are an organization admin or a member of one or more teams, you can select a team from the dropdown. Otherwise, Access defaults to Organization. This selection cannot be changed once the project is saved. 
+1. Click **Save**. 
+
+### Managing Access to Projects
+
+Once a project has been created and saved, the Access (team or organization-wide access) is uneditable. If an organization admin deletes a team from Team Management before managing project access in APIT, the Access defaults to Organization, because Access is required. However, the project will still be available only to the original team and does not change to organization-wide access. Consider using one of the following options to update project access. 
```

Comment:
```suggestion
Once a project has been created and saved, the access level (team or organization-wide access) is uneditable. If an organization admin deletes a team from Team Management before managing project access in API Testing, the access level defaults to Organization because the setting is required. However, the project will still be available only to the original team and does not change to organization-wide access. Consider using one of the following options to update project access. 
```